packages/fxa-settings/src/components/Settings/PageDeleteAccount/index.tsx

```diff
@@ -111,12 +111,17 @@ export const PageDeleteAccount = (_: RouteComponentProps) => {
   const account = useAccount();
 
   const advanceStep = () => {
-    setSubtitleText(
-      l10n.getString('delete-account-step-2-2', null, 'Step 2 of 2')
-    );
-    setConfirmed(true);
+    // Accounts that do not have a password set, will delete immediately
+    if (!account.hasPassword) {
+      deleteAccount('');
+    } else {
+      setSubtitleText(
+        l10n.getString('delete-account-step-2-2', null, 'Step 2 of 2')
+      );
+      setConfirmed(true);
 
-    logViewEvent('flow.settings.account-delete', 'terms-checked.success');
+      logViewEvent('flow.settings.account-delete', 'terms-checked.success');
```

Comment:
Per L86 above:
https://github.com/mozilla/fxa/blob/c7fc7ce8de95342a51423d6fc99446889ab7cadb/packages/fxa-settings/src/components/Settings/PageDeleteAccount/index.tsx#L86

Any issues between alternating "account-delete" and "delete-account"?

Reply:
Hm not sure? I'll probably leave this as is for now 😅